src/config/config.rs

```diff
@@ -267,6 +269,11 @@ pub struct InlineType {
   pub path: Vec<String>,
 }
 
+#[derive(Serialize, Deserialize, Clone, Debug)]
+pub struct AddedFieldType {
```

Comment:
can drop this type after we drop added_field from `Field`. This is only used for generating the blueprint.